execute_test.go

```diff
@@ -42,7 +43,7 @@ func TestRunEcho(t *testing.T) {
 		}
 
 		if result.ExitCode != 0 {
-			t.Error("false should return 1 as its exit code")
+			t.Error("echo should return 0 as its exit code")
```

Comment:
why did you change the name of the function that you called?  this test is calling the bash command "false".